src/resource.js

```diff
@@ -71,12 +72,19 @@ export default class Resource {
    * @returns {String|Array|Object}
    */
   get source() {
```

Comment:
@Fak3 
One of the spec WG meetings has decided to ban any traverses outside of `datapackage.json` basedir for local paths. So we consider it as a rule just waiting official finalization.